### PR TITLE
update remaining maintenance docs

### PR DESCRIPTION
**Pull request by jdmooney**
update single-tier and multi-tier maintenance docs
based on cloudformation doc
fix links in single-tier setup

### CODE REVIEW

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-multi-tier-system.adoc:10]: > Login procedures are described in your setup guide.

Would be good to add a link to the corresponding setup guide.

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc:10]: Would be good to add a link to the corresponding setup guide

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc]:
```suggestion
The system maintains a series of log files that may be helpful when troubleshooting is needed. These logs may be forwarded to Kill Bill to help diagnose your problem.
```

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc]: Is this correct? Shouldn't the Kill Bill logs be `/var/lib/tomcat/logs/killbill.out`?

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left a few comments. Also, the contents of `how-to-maintain-a-single-tier-system.adoc` and how-to-maintain-a-multi-tier-system.adoc` look the same to me. Do we need separate files?

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left a few comments. Also, the contents of `how-to-maintain-a-single-tier-system.adoc` and how-to-maintain-a-multi-tier-system.adoc` look the same to me. Do we need separate files?

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-multi-tier-system.adoc:10]: I agree. I will add the link.

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc:10]: I agree.

Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc:10]: fixed

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-multi-tier-system.adoc:10]: fixed

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: fixed

Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: fixed

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: Both logs exist. Catalina is the more comprehensive. I tried to clarify.

**Comment by jdmooney** [comment]
There are a few differences between the maintain docs for single-tier and multi-tier. The main difference is that the single-tier implementation includes the database and load balancer in the same instance. They could be combined, but there would be times when it would say "this applies to the xxx version only".

**Comment by reshmabidikar** [comment]
> There are a few differences between the maintain docs for single-tier and multi-tier. The main difference is that the single-tier implementation includes the database and load balancer in the same instance. They could be combined, but there would be times when it would say "this applies to the xxx version only".

Got it. Just that, if any updates are required to the docs in the future, both docs would need to be updated consitently. Another idea is to move the common content to a separate file and include it in both files via the `include`directive. See https://github.com/killbill/killbill-docs/blob/v3/userguide/subscription/userguide_subscription.adoc for an example.

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-multi-tier-system.adoc]:
```suggestion
To perform most maintenance tasks you will need to login to your EC2 instances. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide (See https://docs.killbill.io/latest/how-to-set-up-a-multi-tier-system.html#_4_5_login_to_an_instance[4.5. Login to an Instance]).
```

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc]:
```suggestion
To perform most maintenance tasks you will need to login to your EC2 instance. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide (See https://docs.killbill.io/latest/how-to-set-up-a-single-tier-system.html#step_7_login_to_your_instance[Step 7: Login to an Instance])
```

**Comment by reshmabidikar** [comment]
> > There are a few differences between the maintain docs for single-tier and multi-tier. The main difference is that the single-tier implementation includes the database and load balancer in the same instance. They could be combined, but there would be times when it would say "this applies to the xxx version only".
> 
> Got it. Just that, if any updates are required to the docs in the future, both docs would need to be updated consitently. Another idea is to move the common content to a separate file and include it in both files via the `include`directive. See https://github.com/killbill/killbill-docs/blob/v3/userguide/subscription/userguide_subscription.adoc for an example.

I understand that this might be a big change, so let's keep it on hold for now and revisit later.

**Review by reshmabidikar — APPROVED** [review]
Left very minor comments, the PR can be merged once those are addressed.

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-multi-tier-system.adoc]: fixed

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: fixed

**Pull request merged**
